### PR TITLE
Ecdsa verify

### PR DESCRIPTION
**Pull request by LeonMac**
Add a small code for reproduce the testvector verify error.
Total 5 testvectors hardcoded inside. tv1~tv4 pass in rtl testbech test. tv1/tv2 pass in c++ feed test, tv3/tv4 fail in c++ feed test.
tv5 will knock-off the fpga--need reload the fpga afi for recovering normal operation...

### CODE REVIEW

**Comment by LeonMac** [comment]
Merge the openssl lib to generate the test vector dynamically--it is 100% fail, all have 0x8 error report, don't know where the problem would be...

File: aws/cl_zcash/software/runtime/Readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+1. ecdsa_test.cpp use few hardcoded TV for ecdsa signature verify;
+   To compile the ecdsa_test.cpp:
+	make -f makefile_ecdsa
+2. openssl_verify.cpp utilize the openssl lib to generate the test vector dynamiclly
+   To compile the openssl_verify.cpp
+	make -f makefile_ecdsa
```
Comment (bsdevlin) [inline aws/cl_zcash/software/runtime/Readme.md:6]: Is this correct?

File: aws/cl_zcash/software/runtime/ecdsa_test.cpp [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+    size_t len = inStr.length();
+    for (ssize_t i = len-2; i >= 0; i -= 2) {
+        sscanf(inStr.c_str() + i, "%2hhx", outStr);
+        ++outStr;
+    }
+    return true;
+}
+
+
+Signature_t sig_ossl (bool verb)	{	//verb=true, print more info.
```
Comment (bsdevlin) [inline aws/cl_zcash/software/runtime/ecdsa_test.cpp:87]: I think this function (and maybe some other code) is duplicated between files, can you put them into a shared lib or common file?

**Review by bsdevlin — CHANGES_REQUESTED** [review]

File: aws/cl_zcash/software/runtime/Readme.md (hunk 1 above)
Comment (LeonMac) [reply aws/cl_zcash/software/runtime/Readme.md]: NO some typo there, I am going to correct and make a better version.

File: aws/cl_zcash/software/runtime/ecdsa_test.cpp (hunk 2 above)
Comment (LeonMac) [reply aws/cl_zcash/software/runtime/ecdsa_test.cpp]: Will do another version update.

**Pull request merged**